assets/src/js/cookies.js

```diff
@@ -16,36 +16,30 @@ export const setupCookies = () => {
     dataLayer.push(arguments);
   }
 
-  const updateGoogleConsent = (key, granted) => {
+  const updateGoogleConsent = (capabilities) => {
     if (!ENABLE_GOOGLE_CONSENT_MODE) {
       return;
     }
 
-    gtag('consent', 'update', {
-      [key]: granted ? 'granted' : 'denied',
-    });
+    gtag('consent', 'update', capabilities);
     dataLayer.push({
       'event': 'updateConsent',
-      [key]: granted ? 'granted' : 'denied',
+      ...capabilities,
     });
   };
 
-  // If Google Consent Mode is enabled, set default ad storage and analytics storage to 'denied' if needed
-  if (ENABLE_GOOGLE_CONSENT_MODE) {
-    const defaultCookieConsentNeeded = !document.cookie.includes('greenpeace=') && !document.cookie.includes('no_track');
+  const defaultGoogleConsent = () => {
```

Comment:
You could use `updateGoogleConsent` in this function, that would shorten it and avoid some duplication 🙂